source/fuzz/fuzzer_util.h

```diff
@@ -349,19 +349,20 @@ uint32_t MaybeGetStructType(opt::IRContext* ir_context,
                             const std::vector<uint32_t>& component_type_ids);
 
 // Recursive definition is the following:
-// - if |scalar_or_composite_type_id| is a result id of a scalar type - returns
-//   a result id of the following constants (depending on the type): int -> 0,
-//   float -> 0.0, bool -> false.
+// - if |scalar_or_composite_type_id| is a result id of a scalar type -
+// returns
+//   a result id of the following constants (depending on the type): int ->
+//   0, float -> 0.0, bool -> false.
 // - otherwise, returns a result id of an OpConstantComposite instruction.
 //   Every component of the composite constant is looked up by calling this
 //   function with the type id of that component.
 // Returns 0 if no such instruction is present in the module.
-// The returned id either participates in IdIsIrrelevant fact or not, depending
-// on the |is_irrelevant| parameter.
+// The returned id either participates in IdIsIrrelevant fact or not,
+// depending on the |is_irrelevant| parameter.
 uint32_t MaybeGetZeroConstant(
     opt::IRContext* ir_context,
     const TransformationContext& transformation_context,
-    uint32_t scalar_or_composite_type_id, bool is_irrelevant);
+    uint32_t scalar_or_composite_type_id, bool is_irrelevant = false);
```

Comment:
As per my review of your other PR, perhaps there has been a merge problem here?

Reply:
I removed the default value; I can't remember why I introduced it